app/services/platforms/twitch/pagination.ts

```diff
@@ -0,0 +1,3 @@
+export type TwitchPagination = {
+  cursor: string;
+};
```

Comment:
i'm not certain if this deserves it's own file, for interfaces if the main file gets crowded we typically split those out into a `{service}-api` file

Reply:
We will do pagination code here, seemly unrelated to anything else but tags, once we receive feedback from Twitch on their API.

Reply:
ohhh ok i didn't realize this was still WIP when commenting

Reply:
After taking a closer look at this, it doesn't seem like we're gonna be able to decouple pagination enough to not tie it to the specific resource, so I'm having second thoughts about having this file as well.